batch/parsers/testSuiteParserUtils.js

```diff
@@ -0,0 +1,71 @@
+var _ = require('lodash');
+
+export function normalizeTime(rawTime) {
+	let returnValue = 0;
+	if (rawTime) {
+		returnValue = Number.parseFloat(rawTime);
+	}
+	return returnValue.toFixed(2);
+}
+
+function parsedIntOrZero(stringToParse) {
+	return Number.parseInt(stringToParse) || 0;
+}
+export function testSuiteSummary(raw) {
+	return {
+		'tests': parsedIntOrZero(raw.tests),
+		'failures': parsedIntOrZero(raw.failures),
+		'skipped': parsedIntOrZero(raw.skipped),
+		'errors': parsedIntOrZero(raw.errors)
```

Comment:
> Wondering if the [unary `+`](http://stackoverflow.com/questions/17106681/) wouldn't be a better fit in your specific case… if the properties are either a Number or empty String, it could be.